negotiation/Makefile

```diff
@@ -1,6 +1,6 @@
 model=encdec
-exp=basic
-lr=0.5
+exp=decay
+lr=0.031
```

Comment:
you don't have to change the default in the file. you can specify these in cmd line, e.g. `make train lr=0.1`. let's keep the default value in the file.